2-mandatory/6-journey-planner.js

```diff
@@ -119,10 +126,12 @@ function getLocationName() {}
    - Use array method to remove locations that are not accessible by the given transportMode.
    - Use array method to manipulate its elements.
 
-  Advanced challange: try to use arrow function when invoking an array method.
+  Advanced challenge: try to use arrow function when invoking an array method.
 */
 function journeyPlanner(locations, transportMode) {
   // Implement the function body
+  let check = locations.filter(item => item.includes(transportMode))
```

Comment:
How can you rewrite this function using the functions you just implemented above?